import { myError } from './error';

try {
  myError();
} catch (error) {
  console.log('hello!');
  // console.log(error);
}

console.log('test');
